test/lib/jdk/test/lib/security/XMLUtils.java

```diff
@@ -554,7 +554,7 @@ public KeySelectorResult select(KeyInfo keyInfo,
         }
     }
 
-    //////////// MISC ////////////
```

Comment:
This is a section break for various groups of methods in this utility class. Before this there has been
```
    //////////// CONVERT ////////////
    //////////// QUERY AND EDIT ////////////
    //////////// SIGN ////////////
    //////////// VALIDATE ////////////
```
Can you keep the format?

Reply:
Oh, they were recognized as Markdown javadoc? I didn't think of that.

It's strange that the other 4 show no warning, and then I noticed that changing it to
```
    //////////// MISC ////////////

    // Adds a new rule to "jdk.xml.dsig.secureValidationPolicy"
```
can also let the warning go away. The problem is it now treats the `///` line as javadoc and `//` as nothing. This is definitely not right. Maybe I'll do some cleanup later.

Reply:
That said, this is just a test utility class. I'm OK with whatever style at the moment. Thanks.